cmd/agent/agent-local-config.yaml

```diff
@@ -11,7 +10,7 @@ metrics:
       scrape_configs:
         - job_name: local_scrape
           static_configs:
-            - targets: ['127.0.0.1:12345']
+            - targets: ['127.0.0.1:80']
```

Comment:
Are we defaulting the agent to listen to port 80 here, right? I think this means that you'd need root privileges rights on Linux which might not be ideal.